packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -212,19 +208,40 @@ class _SnapshotViewState extends State<SnapshotView> with AutoDisposeMixin {
   }
 
   Widget _buildTreemapAndTableSplitView() {
-    return Split(
-      axis: Axis.vertical,
-      children: [
-        _buildTreemap(),
-        CodeSizeSnapshotTable(rootNode: snapshotRoot),
-      ],
-      initialFractions: const [
-        initialFractionForTreemap,
-        initialFractionForTreeTable,
-      ],
+    return OutlineDecoration(
+      child: Column(
+        children: [
+          areaPaneHeader(
+            context,
+            title: _generateSingleFileHeaderText(),
+            needsTopBorder: false,
+          ),
+          Expanded(
+            child: Split(
+              axis: Axis.vertical,
+              children: [
+                _buildTreemap(),
+                CodeSizeSnapshotTable(rootNode: snapshotRoot),
+              ],
+              initialFractions: const [
+                initialFractionForTreemap,
+                initialFractionForTreeTable,
+              ],
+            ),
+          ),
+        ],
+      ),
     );
   }
 
+  String _generateSingleFileHeaderText() {
+    String output = '';
+    output +=
```

Comment:
declare `output` here instead of declaring it above as the empty string

Reply:
Done